interpreter.go

```diff
@@ -80,6 +80,9 @@ func (intr *treeInterpreter) Execute(node ASTNode, value interface{}) (interface
 	case ASTExpRef:
 		return expRef{ref: node.Children[0]}, nil
 	case ASTFunctionExpression:
+		if _, ok := node.Value.(string); !ok {
+			return nil, errors.New("node value was nil")
```

Comment:
Perhaps report as `invalid node value type` instead (may not be nil, could be an int, etc.)

Reply:
Done